lib/node_modules/@stdlib/blas/ext/base/sdssum/examples/index.js

```diff
@@ -18,19 +18,13 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float32Array = require( '@stdlib/array/float32' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var sdssum = require( './../lib' );
 
-var x;
-var i;
 
-x = new Float32Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( randu()*100.0 );
-}
+var x = filledarrayBy( 10, 'float64', discreteUniform( -100, 100 ) );
 console.log( x );
 
-var v = sdssum( x.length, x, 1 );
-console.log( v );
+var y = sdssum( x.length, x, 1 );
+console.log( y );
```

Comment:
```suggestion
var v = sdssum( x.length, x, 1 );
console.log( v );

```

No need to change the variable name here.